test/unit/spec/EnrollEmail_spec.js

```diff
@@ -46,6 +46,7 @@ define([
       setNextResponse(resp);
       return Util.mockIntrospectResponse(router)
         .then(function () {
+          setNextResponse(resp);
```

Comment:
I don't recall the exact test which showed the issue, I was seeing an "unexpected" xhr request when debugging the test. All these Enroll* tests are doing the same thing (re-setting response after mockIntrospect). My guess is it does not fail usually because the test typically completes before the XHR is requested (sometime after the router is started)